src/Fantomas.Tests/LetBindingTests.fs

```diff
@@ -546,4 +546,10 @@ let ``multiple empty lines between equals and expression`` () =
 
 
     ()
-"""
\ No newline at end of file
+"""
+
+[<Test>]
+let ``should preserve return attributes`` () =
+    formatSourceString false """let f x : [<return: MyCustomAttributeThatWorksOnReturns>] int = x""" config
+    |> should equal """let f x: [<return: MyCustomAttributeThatWorksOnReturns>] int = x
```

Comment:
No space between `:`, see ` ``should handle targets on attributes`` ` test

Reply:
done